askama_derive/src/generator.rs

```diff
@@ -60,7 +60,7 @@ impl<'a> Generator<'a> {
     }
 
     // Takes a Context and generates the relevant implementations.
-    pub(crate) fn build(mut self, ctx: &'a Context<'_>) -> Result<String, CompileError> {
+    pub(crate) fn build(mut self, ctx: &Context<'a>) -> Result<String, CompileError> {
```

Comment:
I relaxed the lifetime rules as they were too strict for my use case. Like this we can pass a temporary `Context` as long as it points to the same data.